common/ripple/userUtils.py

```diff
@@ -43,12 +26,14 @@ async def getPlaytimeTotal(userID: int) -> int:
     """
 
     res = await glob.db.fetch(
-        "SELECT playtime_std, playtime_ctb, playtime_mania, playtime_taiko "
-        "FROM users_stats WHERE id = %s",
+        """
+        SELECT SUM(playtime) AS total_playtime
+        FROM user_stats
+        WHERE user_id = %s
+        """,
         [userID],
     )
-
-    return sum(res.values()) if res else 0
+    return res["total_playtime"] if res else 0

```

Comment:
this will get playtime for vanilla, relax and autopilot modes while previously it was only vanilla - i don't see this as an issue, but was it intended?

Reply:
Fix lol